libclamav/dsig.c

```diff
@@ -256,13 +288,18 @@ int cli_versig(const char *md5, const char *dsig)
     if (strncmp(md5, pt2, 32)) {
         cli_dbgmsg("cli_versig: Signature doesn't match.\n");
         free(pt2);
```

Comment:
Since you added the `goto` error handling, can I trouble you to change it so `pt2` is initalized to NULL and is free'd after the `done` label?

Reply:
done.